tsconfig.json

```diff
@@ -7,6 +7,7 @@
     "emitDecoratorMetadata": true,
     "resolveJsonModule": true,
```

Comment:
That trailing comma trips parsing of tsconfig inside deepkit:

```
Could not parse /home/runner/work/typescript-runtime-type-benchmarks/typescript-runtime-type-benchmarks/tsconfig.json: SyntaxError: Unexpected token } in JSON at position 223
```

Reply:
Ooh I was wondering what was causing for like forever, thanks for catching